package.json

```diff
@@ -4,7 +4,8 @@
   "description": "REPL for the Computable Protocol",
   "main": "index.js",
   "scripts": {
-    "test": "jasmine ./**/*.spec.js"
+    "test": "jasmine ./**/*.spec.js",
+    "start": "node server.js"
```

Comment:
Do we start the repl with `node repl.js`? Maybe we could make that a command also?